batchflow/models/torch/losses/binary.py

```diff
@@ -191,3 +193,27 @@ def forward(self, prediction, target):
         sensitivity = (squared_error * inverse).sum() / (inverse.sum() + self.eps)
 
         return self.r * specificity + (1 - self.r) * sensitivity
+
+
+
+class WeightedBCE(nn.Module):
+    """ Weighted BCE for the unbalanced data which computes weights dynamically """
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, prediction, target):
+        mask = target.float()
+        num_positive = (mask == 1).sum()
+        num_negative = (mask == 0).sum()
+
+        mask[mask == 1] = num_negative / (num_positive + num_negative)
+        mask[mask == 0] = num_positive / (num_positive + num_negative)
+
+        loss = torch.zeros(1, device=prediction.device)
+        loss += F.binary_cross_entropy_with_logits(prediction, target, weight=mask)
+
+        return loss
+
+losses = [WeightedBCE(), Dice()]
+weights = [.5, .5]
+comboloss = Weighted(losses, weights)
```

Comment:
Do you really need it here? There is not a problem to make it in your notebook:

ComboLoss = Weighted([WeightedBCE(), Dice()], [.5, .5])

Nevertheless, `comboloss` here is inappropriate naming.

Reply:
I think, ye, but I thought I should present the full loss used in my models

Reply:
> Do you really need it here? There is not a problem to make it in your notebook:
> 
> ComboLoss = Weighted([WeightedBCE(), Dice()], [.5, .5])
> 
> Nevertheless, `comboloss` here is inappropriate naming.

Why is comboloss inappropriate naming? The bce loss is **combined** with dice loss --> **comboloss**